crates/release-automation/src/lib/crate_selection/mod.rs

```diff
@@ -893,9 +893,10 @@ impl<'a> ReleaseWorkspace<'a> {
                                     insert_state!(CrateStateFlags::HasPreviousRelease);
 
                                     // todo: make comparison ref configurable
-                                    if !changed_files(member.package.root(), &git_tag, "HEAD")?
-                                        .is_empty()
+                                    let changed_files = changed_files(member.package.root(), &git_tag, "HEAD")?;
```

Comment:
This is assigning the changed files to a variable before checking if it's empty. Where is the actual change to check if a dependency of a crate has changed?

Reply:
i added comments to the code in question. PTAL and let me know if that clears it up